File: Java/Fundamentals/src/V_BasicSyntax/T6_Exercise/Exercises/Ages.java
package V_BasicSyntax.T6_Exercise.Exercises;

import java.util.Scanner;

/*
1. Ages
Write a program that determines whether a person is based on the given age:
baby, child, teenager, adult, or elder.

The bounders are:
· 0-2 – baby;
· 3-13 – child;
· 14-19 – teenager;
· 20-65 – adult;
· >=66 – elder;

· All the values are inclusive.
*/
public class Ages {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        //Getting the input
        int n = Integer.parseInt(sc.nextLine());
        //Checking ranges
        if (n < 3) {
            //Print result
            System.out.println("baby");
        } else if (n < 14) {
            //Print result
            System.out.println("child");
        } else if (n < 20) {
            //Print result
            System.out.println("teenager");
        } else if (n < 66) {
            //Print result
            System.out.println("adult");
        } else {
            //Print result
            System.out.println("elder");
        }
        sc.close();
    }
}

File: JavaScript/T1 JS Syntax/ages.js
/*1. Ages
Write a function that determines whether based on the given age a person is: baby, child, teenager, adult, elder.
The input comes as a single number parameter. The bounders are:
· 0-2 (age) – is a baby;
· 3-13 (age) – is a child;
· 14-19 (age) – is a teenager;
· 20-65 (age) – is an adult;
· >=66 (age) – is an elder;
· In all other cases print – "out of bounds";
*/
function determineAge(age) {
    if (age < 0) console.log("out of bounds");
    else if (age < 3) console.log("baby");
    else if (age < 14) console.log("child");
    else if (age < 20) console.log("teenager");
    else if (age < 66) console.log("adult");
    else return console.log("elder");
}



